source/guides/1.1/architecture/interactors.md

```diff
@@ -471,15 +471,140 @@ class AddBook
     @mailer = mailer
   end
 
-  def call(title:, author:)
-    @book = @repository.create({title: title, author: author})
+  def call(book_attributes)
+    @book = @repository.create(book_attributes)
     @mailer.deliver
   end
 end
 ```
 
 Now our Interactor will deliver an email, notifying that a book has been added.
 
+# Integrating With Our Controller
+Finally, we need to call this Interactor from our action.
```

Comment:
Why capital letter?

Reply:
Good point. I was inconsistent in this guide. Just changed them all to lowercase